docs/content/preview/yugabyte-platform/troubleshoot/install-upgrade-issues/kubernetes.md

```diff
@@ -0,0 +1,309 @@
+  ##   Warning  Failed                  22s (x3 over 72s)   kubelet                  Error: ErrImagePull
+  ##   Normal   BackOff                 7s (x3 over 72s)    kubelet                  Back-off pulling image "quay.io/yugabyte/yugaware:2.16.0.0-b90"
+  ##   Warning  Failed                  7s (x3 over 72s)    kubelet                  Error: ImagePullBackOff
+  ```
+
+**Resolution**
+
+- In case of **Bad pull secret / no pull secret / Bad pull secret name** - You need a pull secret to fetch the images from Yugabyte Quay.io and ensure you have applied the same in the namespace, which will use to install YBA. It will default look for the following named `yugabyte-k8s-pull-secret` secret. For more information, see [values.yaml](https://github.com/yugabyte/charts/blob/24a8dcf3a4c33153477e3e3ba82f9f4b6e2967a5/stable/yugaware/values.yaml#L16).
+- For **Unable to pull image** - Ensure the kubernetes nodes have connectivity to Quay.io, or you should have images in the local registry. For more information, see [pull and push yugabytedb docker images to private container registry](https://docs.yugabyte.com/preview/yugabyte-platform/install-yugabyte-platform/prerequisites/#pull-and-push-yugabytedb-docker-images-to-private-container-registry).
+
+
+## Pods are in CrashLoopBackOff
+
+**Symptom**
+
+There can be multiple reasons behind `CrashLoopBackOff`. YBA pod can crash due to some internal application error.
+
+```sh
+## NAME                             READY   STATUS             RESTARTS   AGE
+## yugabyte-platform-1-yugaware-0   3/4     CrashLoopBackOff   2          4d14h
+```
+
+**Debug**
+
+You can use `kubectl` to analyze the situation. We need to check YBA logs to understand the failure.
+
+- Describe the YBA pod facing `CrashLoopBackOff`. Check for `Events`.
+
+  ```sh
+  kubectl describe pods <YBA-POD-NAME> -n <NAMESPACE>
+  ```
+
+- Check the logs for particular container
+
+  ```sh
+  ## yugabytedb anywhere
+  kubectl logs <YBA-POD-NAME> -n <NAMESPACE> -c yugaware
+
+  ## postgres
+  kubectl logs <YBA-POD-NAME> -n <NAMESPACE> -c postgres
+  ```
+
+## CORS error while accessing YBA through LB
+
+You might experience the issue while accessing the YugabyteDB Anywhere through a load balancer. You can define the Cross-Origin Resource Sharing (CORS) domain configuration by setting the [additionAllowedCorsOrigins](https://github.com/yugabyte/charts/blob/master/stable/yugaware/values.yaml#L66) value to the new domain involved. For example, you would add the following to the appropriate Helm command.
+
+  ```properties
+   --set additionAllowedCorsOrigins:'https://mylbdomain'
+  ```
```

Comment:
``````suggestion
**Symptom**
You might experience the issue while accessing the YugabyteDB Anywhere through a load balancer. Initial setup or any login will not work, or it will give you a blank screen.

**Debug**
Check the developer tools of your browser for any errors. Also, check the logs of YugabyteDB Anywhere pod, as follows:

```sh
kubectl logs <POD_NAME> -n <NAMESPACE> -c yugaware
```

```
2023-01-09T10:48:08.898Z [warn] 57fe083d-6ebb-49ab-bbaa-5e6576040d62
AbstractCORSPolicy.scala:311
[application-akka.actor.default-dispatcher-10275]
play.filters.cors.CORSFilter Invalid CORS
request;Origin=Some(https://localhost:8080);Method=POST;Access-Control-Request-Headers=None
```

**Resolution**

Set the correct domain name(s) during Helm install or upgrade by following [Set DNS name](../../../install-yugabyte-platform/install-software/kubernetes/#set-dns-name)
``````